### ISSUE TITLE
Passing `hash` and `outputPath` make builds non deterministic

### ISSUE THREAD

**Issue by Aghassi** [gregberge/loadable-components#897]
https://github.com/gregberge/loadable-components/blob/b3a9febff55b02c49ceba602f07fa5128d53fe36/packages/webpack-plugin/src/index.js#L29

We've found that passing `hash` and `outputPath` make the files generated by this plugin non deterministic. Can we either disable it completely by defaulting to `false` or have a preference to override it so that we can keep builds deterministic? This is because the json file generated will have hashes and full file paths in it, which changes per build. We are using bazel and this means that we can't get cache hits even on almost fully deterministic builds.

CC @joeljeske

**Comment by Aghassi** [comment]
I've opened a PR to address this and start discussions with any maintainers. This issue should avoid being marked as stale or won't fix.

**Comment by Aghassi** [comment]
The issue is going to be fixed, it is waiting on review

**Comment by Aghassi** [comment]
@theKashey can you reopen this and please comment on if this or the PR I opened is unreasonable? It sounded like it could be merged but never did and I'm tired of fighting this bot honestly. I'd like to not maintain my internal patch if this is allowed to be merged.

**Comment by Aghassi** [comment]
@fivethreeo Tagging you too if you'd be interested if @theKashey is busy. I see no reason this can't be upstreamed and would like to work to get it in if so.
